.travis.yml

```diff
@@ -34,7 +34,7 @@ before_script:
   - psql -c 'create user simplified_test;' -U postgres
   - psql -c 'create database simplified_circulation_test;' -U postgres
   - psql -c 'grant all privileges on database simplified_circulation_test to simplified_test;' -U postgres
-  - cd docs && make html && cd ..
+  # - cd docs && make html && cd ..
```

Comment:
This will be good to remove to see if anything breaks on Travis (but can be run locally first to fix any docstring issues).